skyvern-frontend/src/routes/root/Sidebar.tsx

```diff
@@ -1,9 +1,66 @@
-import { Settings } from "./settings/Settings";
+import { cn } from "@/util/utils";
+import {
+  GearIcon,
+  ListBulletIcon,
+  PlusCircledIcon,
+} from "@radix-ui/react-icons";
+import { Link, useLocation } from "react-router-dom";
 
 function Sidebar() {
+  const location = useLocation();
+  let page = "tasks";
+
+  if (location.pathname.includes("create")) {
```

Comment:
should we do exact matching instead?

Reply:
Locations can be dynamic, eventually will include `task_id` and `step_id`, so exact matching is not possible.

But this is not the right way to control the active state of a navigation component, I'll update it to use `NavLink`.